web/src/features/Gitops/components/AppGitops.jsx

```diff
@@ -0,0 +1,310 @@
+import React, { useContext } from "react";
+import Helmet from "react-helmet";
+import CodeSnippet from "@src/components/shared/CodeSnippet";
+import { getAddKeyUri, Utilities } from "../../../utilities/utilities";
+import { useHistory } from "react-router-dom";
+import ConnectionModal from "./modals/ConnectionModal";
+import Loader from "../../../components/shared/Loader";
+import DisableModal from "./modals/DisableModal";
+import { Flex } from "../../../styles/common";
+import { withGitOpsConsumer, GitOpsContext } from "../context";
+import { getLabel } from "../utils";
+import { SERVICES } from "../constants";
+import AppSelector from "./AppSelector";
+
+import "../../../scss/components/gitops/GitOpsDeploymentManager.scss";
+import "../../../scss/components/gitops/GitOpsSettings.scss";
+import "../../../scss/components/gitops/GitopsPrism.scss";
+import { updateAppSList } from "../utils";
+
+const AppGitops = () => {
+  const [ownerRepo, setOwnerRepo] = React.useState("");
```

Comment:
Minor nit - can we import `useState` like we are with `useContext` just for consistency/conciseness?